cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx/Submission.hs

```diff
@@ -855,9 +855,11 @@ txSubmissionClient
         unless (L.null missIds) $
           traceWith bmtr $ TraceBenchTxSubServUnav missIds
         pure $ SendMsgReplyTxs toSend
-          (client done unAcked $
-            stats & over #stsSent        (+ (Sent $ length toSend))
-                  & over #stsUnavailable (+ (Unav $ length missIds)))
+          (client done unAcked $!
```

Comment:
Put the bang on the argument to `client` (one place instead of potentially many) instead of here.

Reply:
..and I missed the other case, indeed.  Thank you!